core/src/main/java/com/linecorp/armeria/client/logging/LoggingClient.java

```diff
@@ -113,6 +116,8 @@ public LoggingClient(Client<I, O> delegate, LogLevel level) {
              level,
              level,
              level,
```

Comment:
So these three above are not used?

Reply:
I forgot to remove it. Thank you @minwoox